README.md

```diff
@@ -8,8 +8,89 @@ User documentation for the SecureDrop Workstation.
 - Run `make docs` to start a live build of the documentation at `http://127.0.0.1:8000`
 - Edit RST files under the `docs` directory - your changes will be reflected in the live build
 
+## Detailed Setup instructions for macOS
+
+There are multiple ways to set your virtual environment; here we will use
+`pyenv` and `pyenv-virtualenv` installed by `brew`.
+
+### Set up pyenv and pyenv-virtualenv
+
+From the Terminal, run
+`brew update && brew upgrade && brew install pyenv pyenv-virtualenv`.
+
+You may want to read more about [pyenv](https://github.com/pyenv/pyenv) and
```

Comment:
This is unnecessary to get the venv up and running, could probably do without it.